components/scroller/scroller_default.story.vue

```diff
@@ -12,6 +12,19 @@
       Go to end
     </button>
 
+    <button @click="addItemTop">
+      Add 10 items top
+    </button>
```

Comment:
The expected behaviour is this:
You might load elements when you are in the position 0 (because you are on the top).
When you load elements you dont want to move the user or the elements (similar when you scroll to top in a chat).
It will load 10 elements and in the same time Im moving the user 10 elements below to the place where the event was triggered.